### PR TITLE
Update browserDetails.version to return int

### LINKED ISSUE #1151: browserDetails.version returns string after PR#1150
- [ ] I have provided steps to reproduce (e.g. a link to a [jsfiddle](https://jsfiddle.net/))
- [X] I have provided browser name, version and adapter.js version
- [X] This issue only happens when adapter.js is used

**Note: If the checkboxes above are not checked (which you do after the issue is posted), the issue will be closed.**

## Versions affected 

**Chrome and Edge Browser all versions

**adapter.js after PR#1150

## Description
browserDetails.version returns string after PR#1150

### PR DESCRIPTION
**Pull request by xingri**
fixes #1151

### CODE REVIEW

File: src/js/utils.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ export function detectBrowser(window) {
       return brand.brand === 'Chromium';
     });
     if (chromium) {
-      return {browser: 'chrome', version: chromium.version};
+      return {browser: 'chrome', version: parseInt(chromium.version)};
```
Comment (fippo) [inline src/js/utils.js]:
```suggestion
      return {browser: 'chrome', version: parseInt(chromium.version, 10)};
```

**Comment by fippo** [comment]
Good catch, thank you!

**Review by fippo — APPROVED** [review]

**Comment by fippo** [comment]
@miketaylr let me guess, this returned the full version because it used to be a string before the UA reduction and now we can't change the API anymore? ;-)

**Comment by miketaylr** [comment]
> and now we can't change the API anymore? ;-)

heh, pretty much. besides, everybody loves strings!

**Pull request merged**